src/test/feature/Makefile

```diff
@@ -1,32 +1,28 @@
 top_builddir = ../../..
 include $(top_builddir)/src/Makefile.global
 
-#.DEFAULT:
-#$(MAKE) -C lib $@
-
-include $(top_builddir)/src/Makefile.global
-
-gtest_include = $(top_builddir)/depends/thirdparty/googletest/googletest/include
-gmock_include = $(top_builddir)/depends/thirdparty/googletest/googlemock/include
-gtest_lib_path = $(top_builddir)/depends/thirdparty/googletest/build/googlemock
-gmock_lib_path = $(top_builddir)/depends/thirdparty/googletest/build/googlemock/gtest
+gtest_include = $(abs_top_srcdir)/depends/thirdparty/googletest/googletest/include
+gmock_include = $(abs_top_srcdir)/depends/thirdparty/googletest/googlemock/include
+gtest_lib_path = $(abs_top_srcdir)/depends/thirdparty/googletest/build/googlemock
+gmock_lib_path = $(abs_top_srcdir)/depends/thirdparty/googletest/build/googlemock/gtest
 
+override CXX = g++
 override CXXFLAGS = -Wall -O0 -g -std=c++11
-override CPPFLAGS := -I/usr/include -I/usr/local/include -I/usr/include/libxml2 -I$(top_builddir)/src/test/feature/ -I$(top_builddir)/src/test/feature/lib/ -I$(top_builddir)/src/interfaces/libpq -I$(top_builddir)/src/interfaces -I$(top_builddir)/src/include  -I$(gtest_include) -I$(gmock_include)
-
+override CPPFLAGS := -I/usr/include -I/usr/local/include -I/usr/include/libxml2 -I$(abs_top_srcdir)/src/test/feature/ -I$(abs_top_srcdir)/src/test/feature/lib/ -I$(abs_top_srcdir)/src/interfaces/libpq -I$(abs_top_srcdir)/src/interfaces -I$(abs_top_srcdir)/src/include  -I$(gtest_include) -I$(gmock_include)
 override LIBS := $(LIBS) -lgtest -lpq -lxml2 -ltest
-override LDFLAGS += -L/usr/local/lib -L/usr/lib -L$(top_builddir)/src/test/feature/lib -L$(gtest_lib_path) -L$(gmock_lib_path)
+override LDFLAGS += -L/usr/local/lib -L/usr/lib -L$(abs_top_srcdir)/src/test/feature/lib -L$(gtest_lib_path) -L$(gmock_lib_path)
 
-PROG = test-main.cpp testlib/test-lib.cpp
+PROG = test-main.cpp $(wildcard testlib/*.cpp)
```

Comment:
It is possible to make the wildcard to include all test folders? i.e., "+PROG = test-main.cpp $(wildcard */*.cpp)"